src/favorite-buttons.ts

```diff
@@ -58,6 +58,7 @@ class FavoriteButtons extends LitElement {
         flex-wrap: wrap;
       }
       .favorite {
+        overflow: hidden;
```

Comment:
What problem did you see when not having this?

Reply:
The background behind the text is a square so if you use a border radius then you would see the corners popping out of the card. Might be better to add a border radius to the favorites title card when I think about it now.